test/e2e/auth/pod_security_policy.go

```diff
@@ -245,6 +260,7 @@ func restrictedPod(name string) *v1.Pod {
 				SecurityContext: &v1.SecurityContext{
 					AllowPrivilegeEscalation: boolPtr(false),
 					RunAsUser:                utilpointer.Int64Ptr(65534),
+					RunAsGroup:               utilpointer.Int64Ptr(65534),
```

Comment:
nit: Can you make 65534 a constant, and replace all uses in this file? ("nobodyUser")